src/pages/billing/NewBillingProjectWizard/GCPBillingProjectWizard/CreateTerraProjectStep.ts

```diff
@@ -0,0 +1,143 @@
+}
+
+export const CreateTerraProjectStep = ({
+  isActive,
+  billingAccounts,
+  onSuccess,
+  ...props
+}: CreateTerraProjectStepProps) => {
+  const [billingProjectName, setBillingProjectName] = useState('')
+  const [chosenBillingAccount, setChosenBillingAccount] = useState<any>()
+  const [isBusy, setIsBusy] = useState(false)
+  const [existing, setExisting] = useState<string[]>([])
+
+  const submit = _.flow(
+    reportErrorAndRethrow('Error creating billing project'),
+    Utils.withBusyState(setIsBusy)
+  )(async () => {
+    try {
+      await Ajax().Billing.createGCPProject(billingProjectName, chosenBillingAccount.accountName)
+      onSuccess(billingProjectName)
+    } catch (error: any) {
+      if (error?.status === 409) {
+        setExisting(_.concat(billingProjectName, existing))
+      } else {
+        throw error
+      }
+    }
+  })
+  return h(Step, { isActive }, [
+    h(StepHeader, {
+      title: 'STEP 4', children: [
+        'Create a Terra project to connect your Google billing account to Terra. ',
+        'Billing projects allow you to manage your workspaces and are required to create one.'
+      ]
+    }),
+    h(StepFields, { disabled: !isActive }, [
+      div([CreateGCPBillingProject({
+        billingAccounts, chosenBillingAccount, setChosenBillingAccount,
+        billingProjectName, setBillingProjectName, existing, disabled: !isActive
+      })]),
+      isActive && _.isEmpty(billingAccounts) ?
+        h(NoBillingAccounts, {
+          refreshed: props.refreshed,
+          setRefreshed: props.setRefreshed,
+          authorizeAndLoadAccounts: props.authorizeAndLoadAccounts
+        }) :
+        div({ style: { display: 'flex', flexDirection: 'column', alignItems: 'center' } }, [
+          h(ButtonPrimary, {
+            onClick: submit,
+            role: 'button',
+            'aria-label': 'create-billing-project',
+            disabled: !isActive
+          }, [
+            'Create Terra Billing Project',
+            isBusy && icon('loadingSpinner', { size: 16, style: { color: 'white', marginRight: '0.5rem' } }),
+          ]),
+          isBusy && div({ role: 'alert', style: { marginTop: '1rem' } }, ['This may take a minute'])
+        ])
+    ])
+  ])
+}
+
+
+const NoBillingAccounts = (props: {
+  refreshed: boolean
+  setRefreshed: (boolean) => void
+  authorizeAndLoadAccounts: () => Promise<void>
+}) => div({
+  style: {
+    display: 'flex', alignItems: 'flex-start',
+    margin: '1rem 1rem 0', padding: '1rem',
+    border: `1px solid ${colors.warning()}`, borderRadius: '5px',
+    backgroundColor: colors.warning(0.10), maxWidth: '45%'
+  },
+  role: 'alert'
+}, [
+  icon('warning-standard', {
+    style: {
+      color: colors.warning(),
+      height: '1.5rem',
+      width: '1.5rem',
+      marginRight: '0.5rem',
+      marginTop: '0.25rem'
+    }
+  }),
+  // TODO: can we get away without the refreshed prop, and just rely on whether or not we've gotten to this step?
```

Comment:
The goal with "refreshed" was that we don't want to keep showing the error each time, and instead what to present them with the UI to contact support once they have attempted to refresh once.